guides/_subsections/upload-an-automated-rule.md

```diff
@@ -0,0 +1,45 @@
+## [Upload an Automated Rule](#upload-an-automated-rule)
+
+Instead of stepping through the form every time, you can also edit your Automated Rule definition files (JSON format) locally and quickly upload them.
+
+Below is an example of a rule file content:
+
+{% highlight bash %}
```

Comment:
Is there a `json` highlight type too? Or is `bash` really the best fit? Even `javascript` might be suitable.